dev/archery/tests/test_benchmarks.py

```diff
@@ -17,26 +17,31 @@
 
 from archery.benchmark.core import Benchmark, median
 from archery.benchmark.compare import BenchmarkComparator
+from archery.benchmark.google import add_unit_to_name, extend_payload
 
 
 def test_benchmark_comparator():
     unit = "micros"
 
     assert not BenchmarkComparator(
         Benchmark("contender", unit, True, [10]),
-        Benchmark("baseline", unit, True, [20])).regression
```

Comment:
Black re-formatted these lines. I can revert the change if you like though I do think this is better ;)